types/jest/jest-tests.ts

```diff
@@ -858,7 +858,7 @@ expect.extend({
 
         const receivedPrinted: string = this.utils.printReceived({});
 
-        const printedWithType: string = this.utils.printWithType('name', {}, (value: {}) => '');
+        const printedWithType: string = this.utils.printWithType('name', {}, (value) => '');
```

Comment:
type in jest is `unknown`, not `any`

Reply:
fixed in https://github.com/facebook/jest/pull/12405, but I think this change is fine anyways